scripts/release_scripts/deploy.py

```diff
@@ -464,6 +463,34 @@ def check_release_doc():
     common.ask_user_to_confirm(message)
 
 
+def update_configs_in_deploy_data(
+        app_name, release_dir_path, deploy_data_path, personal_access_token):
+    """Updates feconf and constants file in deploy data to match the config
+    files in release scripts.
+
+    Args:
+        app_name: str. The name of the app to deploy.
+        release_dir_path: str. Path of directory where all files are copied
+            for release.
+        deploy_data_path: str. Path for deploy data directory.
+        personal_access_token: str. The personal access token for the
+            GitHub id of user.
+    """
+    if app_name == APP_NAME_OPPIASERVER:
+        update_configs.update_feconf_for_main_server(
+            release_dir_path, personal_access_token)
+
+        release_feconf_path = os.path.join(release_dir_path, common.FECONF_PATH)
+        feconf_contents = python_utils.open_file(
+            release_feconf_path, 'r').read()
+        if ('MAILGUN_API_KEY' not in feconf_contents or
+                'MAILGUN_API_KEY = None' in feconf_contents):
+            raise Exception(
+                'The mailgun API key must be added '
+                'before deployment.')
+    update_configs.main(release_dir_path, deploy_data_path)
```

Comment:
What do you think about instead passing a boolean flag to main() that says "prompt_for_mailgun_update"? I think that would be better than having two calls to update_configs, it makes this method read a bit weirdly.

Reply:
Makes sense, done.